runtime/src/main/jni/JsV8InspectorClient.cpp

```diff
@@ -19,7 +20,8 @@ JsV8InspectorClient::JsV8InspectorClient(v8::Isolate* isolate)
       session_(nullptr),
       connection(nullptr),
       context_(),
```

Comment:
It would be wrong to initialize the persistent container with value different than the default `(0)`.